configuration/buildspec.yaml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - yum install gettext -y
+
+  build:
+    commands:
+      #- git clone https://github.com/release-engineering/exodus-lambda.git
+      - cd $CODEBUILD_SRC_DIR
+      #- git checkout deploy
```

Comment:
I think all of this can be removed. Build should already be at root of the repo, exodus-lambda/.